### PR TITLE
Added second last and third last period screen

### LINKED ISSUE #94: Add page to capture past period dates.
Add these two new pages in the app

and

### LINKED ISSUE #100: Fix the duplicate LastPeriodScreen function.
The LastPeriodScreen has repeated code.
Check and remove the duplicated code and fix the bug.

Screenshot 2020-03-25 at 10 57 49 PM

### PR DESCRIPTION
**Pull request by Rits1272**
Fixes #94

### CODE REVIEW

**Comment by Rits1272** [comment]
Please review @sarthology

File: src/screens/ProfileScreen/LanguageScreen.js [hunk 1]
```diff
@@ -60,29 +59,15 @@ export default class LanguageScreen extends Component {
...
         <Image source={top} style={styles.top} />
         <View style={styles.cycleText}>
-          <Text
-            style={[
-              { fontFamily: 'PT-Sans', fontSize: 30, fontWeight: 'bold', alignSelf: 'center' },
-            ]}
-          >
-            please
-          </Text>
-          <Text style={{ fontSize: 25, fontFamily: 'PT-Sans', marginTop: 8, alignSelf: 'center' }}>
```
Comment (sahilit) [inline src/screens/ProfileScreen/LanguageScreen.js:70]: Solve the bug by fixing the font-family, not by removing it.

File: src/screens/ProfileScreen/LanguageScreen.js [hunk 2]
```diff
@@ -60,29 +59,15 @@ export default class LanguageScreen extends Component {
...
-        >
-          language?
-        </Text>
+        <Text style={{ fontSize: 25, alignSelf: 'center', marginBottom: 25 }}>language?</Text>
         <WheelPicker
           selectedItem={this.state.selectedItem}
           data={wheelPickerData}
           onItemSelected={this.onItemSelected}
         />
-        <Image source={please_select_your_language} style={styles.cycleText} />
```
Comment (sahilit) [inline src/screens/ProfileScreen/LanguageScreen.js:85]: Solve the bug by fixing the bug as described in the issue not by removing it.

File: src/screens/ProfileScreen/LastPeriodScreen.js [hunk 3]
```diff
@@ -123,65 +135,4 @@ const styles = StyleSheet.create({
   },
 });
 
-const LastPeriodScreen = () => {
```
Comment (sahilit) [inline src/screens/ProfileScreen/LastPeriodScreen.js:126]: You need to delete the `LastPeriodScreen` upper part not the lower

**Review by sahilit — COMMENTED** [review]
remove the bug fix code. I am assigning it to others

**Comment by sahilit** [comment]
This code has removed essential code changes. so it will not be merged.

**Comment by sahilit** [comment]
You should read the issue before working on it.
You need to solve the issue not just by deleting the code.
It is not a good way of solving bugs for gaining points.
You should also inform before taking the issue.

**Comment by Rits1272** [comment]
I did not see the issues opened until I finally made the pull request. When I pulled the code, these addressed issues were also popping in my screen, so before working on the assigned issue I first had to solve these neccessary bugs so that finally I can test my own code. Its not that I am just doing it for gaining points @sahilit

**Pull request closed without merging**